kolibri/core/assets/src/navigation.vue

```diff
@@ -62,6 +61,20 @@
     },
   };
 
+=======
```

Comment:
More generally this file seems to have gotten into a bad state. Fix up the errors.